### ISSUE TITLE
Tindle Store?

### ISSUE THREAD

**Issue by mermelmadness** [Pixelvation/Pixelvation#2]
I acquired one of these from a friend who got them from PCBWay and had extras.  Love it.  I want to get 5 more but he's not responding to my inquiries.  Do you you have a Tindle store?  I have no experience ordering from a PCB maker.  I've sent companies the gerber files but they keep asking for more information for the ICs and I don't know enough to tell them anything useful.  I'd love to buy them preassembled.  Willing to pay a good amount for them.

**Comment by embedded-creations** [comment]
Hi, join the WLED project discord and look for user Sören. He’s been at least talking about ordering some and reselling

**Comment by mermelmadness** [comment]
I did.  He's who I got the original from.  He did say he ordered a bunch more and was working on the SMD himself, but he's been unresponsive for the last few weeks.  I don't want to keep pestering him in case he's going through some personal stuff that's keeping him away, so I looked into having them printed myself.

**Comment by mermelmadness** [comment]
I've been trying to submit the gerber, BOM, and CPL files to PCB printers, but either they don't process correctly in their system or they don't have all the components required.  Soren is still unresponsive, so I'm currently stuck.  Really want more Pixelvation boards.

**Comment by embedded-creations** [comment]
Did you try JLCPCB? That’s where I ordered from and they’re most likely to have all the parts

**Comment by mermelmadness** [comment]
JLCPCB was the first place I tried.  They don't recognize one of the components from the BOM and don't have enough of another.  When searching for it, I found a similar part, but it seems to be out of stock.
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/2ac5533e-0dd1-437b-a35c-dabdc6dca566)
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/8689f096-7e87-49b6-9220-8e4b8bb63531)

**Comment by mermelmadness** [comment]
Apologies, but I haven't had any luck with any PCB Printing service.  They all say that the BOM and POS file are incomplete with part info.  I'm sorry that I don't know enough about this to handle this myself.  I just really wanted to buy 5 of the boards to build a 3x2 information panel for my office (192x128).

**Comment by mermelmadness** [comment]
@embedded-creations ok, I got them.  Time to do some soldering.  Thank you for your help and permission to print these for myself.  Keep an eye on the WLED Discord for project updates in the coming weeks.
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/a3aee434-d4a3-4afb-a9cb-b42a8ea27c04)

**Comment by embedded-creations** [comment]
Sweet, thanks for the update. Any advice for others trying to order? Did you find a substitute for the missing part?

**Comment by mermelmadness** [comment]
From what I went through, the BOM lists the pinholes as parts, so they will come up as not found when you submit it.

The out of stock part was C223608
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/8c82f4ac-4c99-4c17-84cf-1b328db56bab)

I replaced it with C8598
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/6065cfba-f0a0-4359-83da-1b84ea3cf26b)

I had to go to a forum to ask what replacement parts would be viable and they mentioned the LCSC product numbers.

Also, the original rendering of the assembled board had the voltage regulator offset by 90 degrees.
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/a0c9d606-0221-4c75-819e-fb447b060c3f)

Thankfully their engineers know what they're doing, so they corrected for it.
![image](https://github.com/Pixelvation/Pixelvation/assets/38080873/4e5a4dac-f47f-400b-aab0-3cc317bd42ad)

I'll be soldering the pins tonight and probably flashing over the weekend.  Gotta remember how to use VSCode and PlatformIO
